### ISSUE TITLE
Fail to exit application, Windows 11, local mode

### ISSUE THREAD

**Issue by jaxsonpd** [jaxsonpd/better-serial#2]
# Failed to exit on windows

**OS:** Windows 11
**mode:** local

---
When running in local mode on windows the application fails to return to the command prompt when completed. The following error is created:

```
Exception in thread Thread-2:
Traceback (most recent call last):
  File "threading.py", line 932, in _bootstrap_inner
  File "threading.py", line 870, in run
  File "com_tx.py", line 92, in com_tx_local_thread_entry
EOFError
Traceback (most recent call last):
  File "main.py", line 96, in <module>
  File "main.py", line 92, in main
  File "threading.py", line 1011, in join
  File "threading.py", line 1027, in _wait_for_tstate_lock
KeyboardInterrupt
[5424] Failed to execute script 'main' due to unhandled exception!
```

**Comment by jaxsonpd** [comment]
The exit system has to be completely re-writen to stop these kind of errors from occuring.

**Comment by jaxsonpd** [comment]
This issue has been some what addressed by pull request #4 but remains open.

**Comment by jaxsonpd** [comment]
This has been fixed
